File: Vinayak Sharma/Day-4/4A.java
class Solution {
    public int removeDuplicates(int[] nums) {
        int k=0;
        for(int i=0; i<nums.length-1; i++){
            if(nums[i] == nums[i+1]){
                k++;
            }else{
                nums[i-k+1]=nums[i+1];
            }
        }
        return nums.length-k;
    }
}

File: Keshav Banka/Day 4/4A.cpp
// Question: https://leetcode.com/problems/remove-duplicates-from-sorted-array

// Remove Duplicates from sorted array

class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        int i = 0, n = nums.size();
        for (int j = 0; j < n; j ++) {
            int temp = nums[j];
            swap(nums[i], nums[j]);
            i ++;
            while (j < n-1 && temp == nums[j+1])  j ++;
        }
        return i;
    }
};
